docs/configuration.md

```diff
@@ -6,14 +6,14 @@ Pyright settings can also be specified in a `[tool.pyright]` section of a “pyp
 
 Relative paths specified within the config file are relative to the config file’s location. Paths with shell variables (including `~`) are not supported. Paths within a config file should generally be relative paths so the config file can be shared by other developers who contribute to the project.
 
-## Main Configuration Options
+## Environment Options
```

Comment:
I really like the way you've (re)organized these options. These headings provide a lot of clarity.